tasks/managed/collect-push-rpm-params/README.md

```diff
@@ -0,0 +1,10 @@
+# collect-push-rpm-params
+
+Tekton task to get params for task(push-rpm-to-koji) of pushing konflux build rpms to koji instance.
+
+## Parameters
+
+| Name                 | Description                                                               | Optional | Default value |
+|----------------------|---------------------------------------------------------------------------|----------|---------------|
+| dataPath             | Path to the JSON file of the merged data to use in the data workspace   | No       | -             |
```

Comment:
looks like you need 2 space added to the end of the description to make the |'s vertically align